data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+					decline
+			label accept
+			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refuelling drills, it would be."`
+			`	She tells you she'll pay you <payment> when you reach <planet>, and will prepare another "drill" for you to help with too. She heads back inside one of the Judicators, and they prepare to follow your ship when you launch.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Heliarch"
+		fleet
+			names "heliarch"
+			fighters "heliarch fighter"
+			variant
+				"Heliarch Judicator" 2
+				"Heliarch Pursuer" 4
+				"Heliarch Rover" 4
+				"Heliarch Stalker" 4
+	on visit
+		dialog `You have reached <planet>, but you left some of the Heliarch escorts behind. Better depart and wait for them to get here.`
+	on complete
+		"coalition jobs" ++
+		payment 150000
+		conversation
+			`Lotuora pays you <payment> when you land. "An idea for the next drill, I've had," she says. "Need to arrange some more ships, I will, so some hours, I will need. Meet me in the spaceport when I'm done, you should."`
+
+mission "Heliarch Drills 2-A"
+	name "Pursuit Drill"
+	description "Land on the <planet> while avoiding the Heliarch Hunters."
+	source "Belug's Plunge"
+	destination "Shadow of Leaves"
+	to offer
+		has "Heliarch Drills 1: done"
+		not "Resistance: Questions: done"
+	on offer
+		conversation
+			`When you find Magister Lotuora in the spaceport, she shows you to a map of the Coalition, pointing to two systems, one in Kimek and one in Saryd space. "Made many calls, I have, and to assemble many ships for this drill, I've managed. How fit for their job, our Hunters are, we shall see."`
+			choice
+				`	"What do you mean?"`
+					goto explain
+				`	"Will I be fighting them?"`
+			`	"No, no fighting," she says. "Not yet. Only simulating a pursuit, they will be."`
+			label explain
+			`	She points to the systems again, "To land on <planet> and Ashy Reach unseen by the Hunters, your task is. Attack you, they will not, but allow them to be in the system when you land, you cannot. Once cleared both planets, you have, return here for 300,000 credits, you may."`
+			choice
+				`	"Ok, I'll head out to the planets then."`
+					goto accept
+				`	"Can you tell me where my pursuers will be?"`
+			`	"Know of their pursuers, the pursued do not," she says, letting a laugh as if it was a joke. "Well, usually... From many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
+			label accept
+			`	She points again to one of the planets, "Still preparing, some Hunters around Ashy Reach are, so head for <planet> first, you should," she wishes you luck on avoiding the Heliarch Hunters, and heads off to a Heliarch military building.`
+				accept
+	npc evade
+		government "Heliarch"
```

Comment:
I wish there was some way to tell these Hunters apart, like a slightly different color or smth like that, because rn I can't make them apart, and just ended up landing with one each time without realizing it, and then I randomly jump drived away to come back.

Reply:
I'll make them `target`.